components/why-kedro/why-kedro.module.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: $max-content;
+  overflow: hidden;
+  text-align: center;
+  width: 100%;
+}
+
+.title {
+  @include h1;
+
+  margin-bottom: 3rem;
+}
+
+.tabsWrapper {
+  margin: 0 auto 2.625rem;
+
+  @include breakpoint(small) {
+    margin: 0 auto 1.375rem;
+    padding-left: 1rem;
+    width: 100%;
+  }
+}
+
+.list,
+.listItem {
```

Comment:
as discussed in our last PR, given that we are still using sass, moreover with how we declared our variables, let's follow the hyphen-delimited approach to naming our scss casses.

Reply:
It does seem we can keep it how it's written here based on [Next's own documentation](https://nextjs.org/docs/basic-features/built-in-css-support#sass-variables). I'm trying to follow their example.

Were you able to look at the rest of the PR? Would love if you can please approve/request changes before the EOD day if possible!

Reply:
I understand we can keep it, but that doesn't mean we should follow it - I still think there is merit in maintaining the consistency of the css naming convention we have employed in the team, moreover it being in line with the variable naming that was set up in `variables.scss`. 

I had put in my review earlier, let me know if there's anything unclear. I can come back to review this later today once the requested changes are made to help move this forward.

Reply:
You can review it again now. I pushed some changes ~5 minutes ago for the style fixes you caught.

Reply:
I'm trying to draw a distinction between the code and product of Kedro-Viz and this Kedro website code and project. I don't believe all conventions in one need to be transferred over to the other.

If the logic is that we should align with the variable naming in `variables.scss` then I think that means we actually **do** want to use camelCase. This is the pattern and best practice outlined by the framework.

Next has this:

```scss
$primary-color: #64FF00

:export {
  primaryColor: $primary-color
}
```

and then they reference it like so:

```jsx
<Layout color={variables.primaryColor} />
```

Is that what we're doing as well?

Reply:
hhmm I see - personally I love keeping style classes as kebab case as I often see in most FE projects, but I understand if this is the norm for next.js. In this case it's fine and it's no fuss - we can keep it as is then.